src/components/tabs/FunctionsTab.vue

```diff
@@ -95,6 +117,14 @@ export default class FunctionsTab extends Vue {
     return [];
   }
 
+  private getFileIndexFromFilename(filename: string) {
+    return this.files.findIndex((file) => file.filename === filename);
+  }
+
+  private getFunctionIndexFromFunctionName(fileIndex: number, functionName: string) {
+    return this.files[fileIndex].functions.findIndex((func) => func.name === functionName);
+  }
```

Comment:
It makes more sense to me to have these as getters so that they can be reused elsewhere

Reply:
True! I've moved those to the `getters.ts` file.